frappe/tests/test_translate.py

```diff
@@ -71,10 +71,31 @@ def test_read_language_variant(self):
 		try:
 			frappe.local.lang = "pt-BR"
 			self.assertEqual(_("Mobile No"), "Telefone Celular")
+			frappe.local.lang = "pt"
+			self.assertEqual(_("Mobile No"), "Nr. de Telemóvel")
+		finally:
+			frappe.local.lang = "en"
+			self.assertEqual(_("Mobile No"), "Mobile No")
+
+	def test_write_language_variant(self):
+		def import_export_translation(lang, updates):
+			path = frappe.get_app_path("frappe", "translations", lang + ".csv")
```

Comment:
It wasn't lost, I removed it because this has same original bug. It doesn't read the data right :) 

![image](https://github.com/frappe/frappe/assets/9079960/288bbf57-d3bf-43fd-935f-d5dbc9b97fd9)

Reply:
@ankush: Right, we settled with HarryPaulo‘s version 4203c3b, so the test had to be adapted, right?
Thank you for caring so much! 🙏

I think the test should‘ve explicitly failed though. We might want to test a second translated word and make sure it’s not lost.
But then again, we may also be happy with what we have, relax, and go on… 😎